core/state/state_object.go

```diff
@@ -247,7 +273,7 @@ func (s *StateObject) GetCommittedState(db Database, key common.Hash) common.Has
 			*meter += time.Since(readStart)
 			readStart = time.Now()
 		}
-		if metrics.EnabledExpensive {
+		if metrics.EnableIORecord {
```

Comment:
Whey we change the flag? It will be difficult for others to use our metrics. They need to resolve such conficits.

Reply:
it is unuseful in lastest code

Reply:
I will delete this change later